WalletWasabi/BitcoinCore/Processes/BitcoindRpcProcessBridge.cs

```diff
@@ -60,7 +60,7 @@ public async Task StartAsync(CancellationToken cancel)
 
 				if (process is null || process.HasExited)
 				{
-					throw ex;
+					throw new Exception($"Failed to start daemon, location: '{process?.StartInfo.FileName} {process?.StartInfo.Arguments}'", ex);
```

Comment:
Can you create a PR with this beautification?

Reply:
Done #3218